skimage/measure/_marching_cubes_lewiner.py

```diff
@@ -57,6 +57,11 @@ def marching_cubes_lewiner(volume, level=None, spacing=(1., 1., 1.),
         produce a topologically correct result. The results with using
         this option are *not* generally the same as the
         ``marching_cubes_classic()`` function.
+    mask : (M, N, P) array
+        Boolean array. The marching cube algorithm will be computed only on
+        True elements. This will save computational time when interfaces
+        are located within certain region of the volume M, N, P and also
+        allow to compute finite surfaces.
```

Comment:
I am not sure what a "finite surface" is: could you use a different phrasing or, if this is a technical term, please explain it in the docstring?

Reply:
I think the story here is unclear: "when interfaces are located within certain regions"—what regions?

Reply:
I just added an example of what I mean by certain region and added an explanation of the finite surface concept